_incObj/sub CalcSine.asm

```diff
@@ -24,39 +24,6 @@ CalcSine:
 
 ; ===========================================================================
 
-Sine_Data:	incbin	"misc\sinewave.bin"	; values for a 360� sine wave
+Sine_Data:	incbin	"misc\sinewave.bin"	; values for a 360º sine wave
```

Comment:
The line uses º, which is apparently [an ordinal indicator](https://en.wiktionary.org/wiki/%C2%BA); perhaps this should be °, [the degree sign](https://en.wiktionary.org/wiki/%C2%B0)?